js/src/elements/form-behavior.ts

```diff
@@ -364,6 +366,16 @@ export class FormBehaviorElement extends HTMLElement {
 		return null;
 	}
 
+	validateGroupRequired(input: HTMLValueElement, label: string): string | null {
+		const group = getField(input);
+		const inputs = Array.from(group.querySelectorAll<HTMLInputElement>('input'));
+		const selected = inputs.some(input => input.checked);
```

Comment:
Super nit, feel free to ignore:

The variable name here makes me think I'd get a reference to a checked/selected input. Instead, we're going for more of a boolean as to whether there is an active selection.

What about `hasCheckedInput`?

Reply:
✅